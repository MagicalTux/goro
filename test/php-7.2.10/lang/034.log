
---- EXPECTED OUTPUT
3,14
---- ACTUAL OUTPUT
Call to undefined function setlocale in /home/ronald/proj/goro/test/php-7.2.10/lang/034.php on line 3
---- FAILED
