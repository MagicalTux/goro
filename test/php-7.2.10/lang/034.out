Call to undefined function setlocale in /home/ronald/proj/goro/test/php-7.2.10/lang/034.php on line 3